pallets/pools/src/tranche.rs

```diff
@@ -1453,6 +1490,40 @@ where
 	Ok(risk_buffers)
 }
 
+pub(crate) fn create_asset_metadata<Balance, CurrencyId>(
+	decimals: u32,
+	currency: CurrencyId,
+	parachain_id: ParaId,
+	token_name: Vec<u8>,
+	token_symbol: Vec<u8>,
+) -> Result<AssetMetadata<Balance, CustomMetadata>, DispatchError>
+where
+	Balance: Zero,
+	CurrencyId: Encode,
+	CustomMetadata: Parameter + Member + TypeInfo,
+{
+	let tranche_id = currency.encode();
+
+	Ok(AssetMetadata {
+		decimals,
+		name: token_name,
+		symbol: token_symbol,
+		existential_deposit: Zero::zero(),
+		location: Some(VersionedMultiLocation::V1(MultiLocation {
+			parents: 1,
+			interior: X2(Parachain(parachain_id.into()), GeneralKey(tranche_id)),
+		})),
+		additional: CustomMetadata {
+			mintable: false,
+			permissioned: false,
```

Comment:
```suggestion
			permissioned: true,
```

Reply:
I don't think we should, since we will likely add a custom token permission check in restricted-tokens at some point, which applies to tokens which have this flag set to `true`. We will then also still have the tranche token specific permission checks. This would mean that we are checking the permissions for a tranche token twice.

Reply:
> We will then also still have the tranche token specific permission checks
I hear you, but those are not mutually exclusive properties, as we can still have a tranche-specific check. For example:

``` rust
if metadata.additional.permissioned {
  match currency {
    CurrencyId::Tranche(_, _) => specific_tranche_check(...),
    _ => default_check(...),
  }
}
```

This `permissioned` property was initially added to mitigate the requirement of flagging the permission requirements of tranche tokens, so if we now set this to `false` it sort of defeats the purpose. Which is fine if we no longer want it, but it seems off to me to have two sources of truth for permission flagging (one with this field, the other being implicit throughout the code). What are your thoughts?